altair/expr/__init__.py

```diff
@@ -117,6 +117,115 @@ class expr(_ExprRef, metaclass=_ExprMeta):
+    .. _Number.isFinite:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite
+    .. _Math.abs:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/abs
+    .. _Math.acos:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/acos
+    .. _Math.asin:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/asin
+    .. _Math.atan:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan
+    .. _Math.atan2:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/atan2
+    .. _Math.ceil:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/ceil
+    .. _Math.cos:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/cos
+    .. _Math.exp:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/exp
+    .. _Math.floor:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/floor
+    .. _Math.hypot:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/hypot
+    .. _Math.log:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/log
+    .. _Math.max:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/max
+    .. _Math.min:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/min
+    .. _Math.pow:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/pow
+    .. _Math.random:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+    .. _Math.round:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/round
+    .. _Math.sin:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sin
+    .. _Math.sqrt:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/sqrt
+    .. _Math.tan:
+       https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math/tan
+    .. _normal (Gaussian) probability distribution:
+       https://en.wikipedia.org/wiki/Normal_distribution
+    .. _cumulative distribution function:
+       https://en.wikipedia.org/wiki/Cumulative_distribution_function
+    .. _probability density function:
+       https://en.wikipedia.org/wiki/Probability_density_function
+    .. _log-normal probability distribution:
+       https://en.wikipedia.org/wiki/Log-normal_distribution
+    .. _continuous uniform probability distribution:
+       https://en.wikipedia.org/wiki/Uniform_distribution_(continuous
```

Comment:
I think this is broken also

Reply:
good catch, this was broken due to the markdown parser being confused by the closing paren in the URL. Fixed by adding applying replacements to the string before parsing as markdown in [4433faa](https://github.com/vega/altair/pull/3699/commits/4433faa15eb87c93ef82d7f0c531c6b625801814)